modules/refined/src/test/scala/doobie/refined/RefinedSuite.scala

```diff
@@ -54,15 +52,18 @@ class RefinedSuite extends munit.FunSuite {
   }
 
   test("Query should return a refined type when conversion is possible") {
-    sql"select 123".query[PositiveInt].unique.transact(xa).void.unsafeRunSync()
+    sql"select 123".query[PositiveInt].unique.transact(xa)
+      .assertEquals(Refined.unsafeApply[Int, Positive](123))
```

Comment:
Ah, that's right: there's no macro support in Refined for Scala3, unfortunately.

Just in case, there's another form for create refined values in runtime that you may find less cumbersome:
```suggestion
      .assertEquals(refineV[Positive].unsafeFrom(123))
```
But that one you are using works too 👍